### PR TITLE
[UK] Added Great! TV, fixed STV and added Quest

### PR DESCRIPTION
**Pull request by ExperiencersInternational**
Self-explanatory.

Links taken from submissions to my repo at https://github.com/ExperiencersInternational/tvsetup/blob/main/config.m3u

Solves #12460 and #12446

### CODE REVIEW

File: streams/uk.m3u [hunk 1]
```diff
@@ -165,9 +169,9 @@ https://ayozat-live.secure2.footprint.net/egress/bhandler/ayozat/sportystufftv/m
 #EXTINF:-1 tvg-id="SportyStuffTV.uk" status="online",Sporty Stuff TV (720p)
 https://ayozat-live.secure2.footprint.net/egress/bhandler/ayozat/sportystufftv/playlist.m3u8
 #EXTINF:-1 tvg-id="STV.uk" status="timeout",STV (1080p) [Geo-blocked]
-https://csm-e-stv.tls1.yospace.com/csm/live/139900483.m3u8
+https://csm-e-stv-eb.tls1.yospace.com/csm/live/139900483.m3u8?yo.l=true&yo.ls=2,3,4,5,6&yo.p=3&ref=a1c9d4c18ab942e779c3b0fb86ebf4fdbcf81cad3cfc56b066c275621098fc84&s=
```
Comment (BellezaEmporium) [inline streams/uk.m3u:172]:
```suggestion
https://csm-e-stv.tls1.yospace.com/csm/live/139900483.m3u8?yo.l=true&yo.ls=2,3,4,5,6&yo.p=3
```

This is sufficient to make it work, from my testings.

File: streams/uk.m3u [hunk 2]
```diff
@@ -165,9 +169,9 @@ https://ayozat-live.secure2.footprint.net/egress/bhandler/ayozat/sportystufftv/m
...
 #EXTINF:-1 tvg-id="STVPlus1.uk" status="timeout",STV+1 (1080p) [Geo-blocked]
-https://csm-e-stv.tls1.yospace.com/csm/live/181023311.m3u8
+https://csm-e-ces1aeuw1live102-0ad3dd531b3c30bf5.tls1.yospace.com/csm/live/181023311.m3u8?yo.l=true&yo.ls=2,3,4,5,6&yo.p=3&ref=a1c9d4c18ab942e779c3b0fb86ebf4fdbcf81cad3
```
Comment (BellezaEmporium) [inline streams/uk.m3u:174]: https://csm-e-stv.tls1.yospace.com/csm/live/181023311.m3u8?yo.l=true&yo.ls=2,3,4,5,6&yo.p=3&

Comment (BellezaEmporium) [inline streams/uk.m3u:174]:
```suggestion
https://csm-e-stv.tls1.yospace.com/csm/live/181023311.m3u8?yo.l=true&yo.ls=2,3,4,5,6&yo.p=3
```

**Review by BellezaEmporium — APPROVED** [review]
tested through an online proxy based in the UK for the other links, answers correctly.

**Review by freearhey — APPROVED** [review]

**Pull request merged**